api/controllers/search.js

```diff
@@ -454,20 +455,47 @@ var searchCollection = async function (roles, keywords, schemaName, pageNum, pag
     ]
   };
 
+
+  
   console.log("modifier:", modifier);
   console.log("match:", match);
 
-  var sortingValue = {};
-  sortingValue[sortField] = sortDirection;
+
+  if (schemaName == "Document"){
+    twoSorts = true;
+  }else {
+    sortingValue = {};
```

Comment:
A good rule of thumb to try and avoid modifying any parameters. I might try creating a new variable to handle these values.